decidim-dev/lib/decidim/dev/test/rspec_support/capybara.rb

```diff
@@ -62,7 +62,7 @@ def switch_to_default_host
   end
 
   config.around :each, :slow do |example|
-    max_wait_time_for_slow_specs = 12
+    max_wait_time_for_slow_specs = 30
```

Comment:
did you find problems with conversation related tests? where did you find the problems?

Reply:
Failures were related to comments, I think, and increasing this value helped fix the tests...